lib/helpers.js

```diff
@@ -33,11 +33,9 @@ _beforeEach.cleanDatasource = function(dsName) {
     if (typeof this.app === 'function'
         && typeof this.app.datasources === 'object'
         && typeof this.app.datasources[dsName] === 'object') {
-      this.app.datasources[dsName].automigrate();
+      this.app.datasources[dsName].automigrate(done);
       this.app.datasources[dsName].connector.ids = {};
     }
-    
-    done();
```

Comment:
I think you should still call `done` in the case when the condition on lines 33-35 is false, otherwise the callback is never called.

Other than that, this looks good to me.